main.py

```diff
@@ -25,6 +25,32 @@ def __contains__(self, item):
     def __iter__(self):
         yield self
 
+def validate_stability(Kp, Ki):
+    if (((Kp + Ki)*(Kp + Ki)) < (4*Ki)):
```

Comment:
What's the reason of adding this condition?

Reply:
the goal is to skip unstable parameters and potentially replace them by a different pair